advancedblacklist/commands/whitelist.py

```diff
@@ -0,0 +1,180 @@
+# Copyright (c) 2021 - Jojo#7791
```

Comment:
Please see my review comments from `advancedblacklist/commands/blacklist.py`. Those changes should also be reflected in this file if you wish to make them.